project_generation/content/templates/event/service/service.go.tmpl

```diff
@@ -104,7 +104,7 @@ func (svc *Service) Close(ctx context.Context) error {
 		// The kafka consumer will be closed after the service shuts down.
 		if svc.serviceList.KafkaConsumer {
 			log.Info(ctx, "stopping kafka consumer listener")
-			if err := svc.consumer.StopListeningToConsumer(ctx); err != nil {
+			if err := svc.consumer.StopAndWait(); err != nil {
```

Comment:
I believe we should just use the Stop method instead of StopAndWait

Reply:
From the readme , it seems the stopandwait as the replacement for StopListeningToConsumer. please let me know if this is not the case.

Reply:
done.